### ISSUE TITLE
Is there a composition error?

### ISSUE THREAD

**Issue by Feywell** [Yaoyi-Li/GCA-Matting#19]
I try to use composition loss in my training.
And then, I find the merged img's color is different from image.
Why the color jitter is just be used on fg? Is it the reason for color difference?

By the way, I find regression_loss just use unkonwn area. 
It will be work well when the composition loss used in unknowd area?

**Comment by Yaoyi-Li** [comment]
It is because there are 100k+ bg images in COCO but only 400+ fg. And some of the jitters on merged images will probably be neutralized by batchnorm.

In our codes the composition loss is calculated in unknown area as proposed in the paper of Deep Matting. Since the composition loss is not a part of our method, we did not conduct many experiments on it.
